viz_scripts/scaffolding.py

```diff
@@ -69,6 +69,16 @@ def load_all_participant_trips(program, tq, load_test_users):
     disp.display(participant_ct_df.head())
     return participant_ct_df
 
+def filter_composite_trips(all_comp_trips, program, load_test_users):
```

Comment:
I see that `filter_composite_trips` is not identical to `filter_labeled_trips` because it doesn't filter by blank user input. However, I don't understand why you are not filtering by blank user input. It should be possible to just call `load_all_participant_trips` instead of `load_all_confirmed_trips` followed by `filter_composite_trips` to achieve the same result.

Reply:
>However, I don't understand why you are not filtering by blank user input.

I do, later, but I need all of the trips in order to create the "all trips for which a survey was prompted" information

Reply:
I made some changes to the data loading (including pushing it to scaffolding) in #135 - I have the filtered, unfiltered, and file suffix all returned from a function in scaffolding now - I don't have the debug_df or quality_text included since it varies chart to chart